src/apiary.apib

```diff
@@ -1544,6 +1544,81 @@ level `conversions` array.
                 + conversion (Conversion)
                     + conversion_type: 3 (number)
 
+
+# Group Alert
+
+Alerts are used across application to notify users about events that are triggered in the platform. Notifications are sent to users that have subscribed to an alert using a selected channel
+
+## Alert [/alerts/alert]
++ Attributes
+    + alert_id: 23 - ID of the alert
+    + code: Example -  Code used internally for reference
+    + name: Example alert - Display name of an Alert
+    + description: This alert will notify you about particular event (nullable, string) -  Fired when new campaign is created
+    + service: ExampleAlert (string) - Name of a service in DIC
```

Comment:
Can we abstract the DIC? It wouldnt make much sense to an external client using these API docs?